source/client/TableView.py

```diff
@@ -133,7 +133,7 @@ def textObjects(self, text, font, color):
 
     def Network_publicInfo(self, data):
 
-        # TODO: update example below.  False should be a play state, not True/False
+        # todo: update example below.  False should be a play state, not True/False
```

Comment:
todo should stay all caps - thats the convention for a reason. If you specifically want this todo not to be highlighted just make it a comment with out the keyword (I know this is nit-picky, but its the only change here so it stood out).